test/tesla/middleware/fuse_test.exs

```diff
@@ -67,7 +68,7 @@ defmodule Tesla.Middleware.FuseTest do
   end
 
   test "custom should_melt function - not melting 500" do
-    custom_client = ClientWithCustomShouldMeltFunction
+    custom_client = ClientWithCustomSetup
```

Comment:
Let's get rid of this variable completely, it's quite confusing.

Reply:
Sure!